tests/test_stream.py

```diff
@@ -0,0 +1,41 @@
+from lomond.stream import WebsocketStream, FrameParser
+from lomond.errors import ProtocolError
+import pytest
+
+
+@pytest.fixture
+def stream():
+    return WebsocketStream()
+
+
+def test_constructor(stream):
+    assert isinstance(stream, WebsocketStream)
+
+
+def test_feed(stream):
+    data = (
```

Comment:
Could we test a few different combinations of the chop size in the data. For instance, feeding a byte at a time, 2 bytes at a time, line at a time etc.

You might only be testing the Parser class indirectly, but I could see this catching some subtle flaw.